node/src/tests/inherent_data_tests.rs

```diff
@@ -20,24 +22,28 @@ async fn block_proposal_cidp_should_be_created_correctly() {
 
 	let inherent_data_providers = ProposalCIDP::new(
 		test_create_inherent_data_config(),
-		test_client(),
+		TestApi::new(ScEpochNumber(2))
+			.with_headers([(H256::zero(), mock_header())])
+			.into(),
 		TestDataSources::new().into(),
 	)
 	.create_inherent_data_providers(H256::zero(), ())
 	.await
 	.unwrap();
 
 	#[cfg(not(feature = "block-beneficiary"))]
-	let (slot, timestamp, mc_hash, ariadne_data) = inherent_data_providers;
+	let (slot, timestamp, mc_hash, ariadne_data, native_token) = inherent_data_providers;
```

Comment:
Side note: why are we having `block-beneficiary` feature?

Reply:
I think this is just a left-over. I guess we can remove it, I'll push a separate PR for that.

Reply:
https://github.com/input-output-hk/partner-chains/pull/50